data/nemo.py

```diff
@@ -177,24 +173,6 @@ def __latlon_vars(self, variable):
 
         raise LookupError("Cannot find latitude & longitude variables")
 
-    def get_path(self, path, depth, time, variable, numpoints=100, times=None, return_depth=False):
```

Comment:
I've deleted `get_path` from fvcom, nemo, and mercator and moved it into model, since the implementations were identical.